test/parallel/test-vm-basic.js

```diff
@@ -128,3 +128,23 @@ const vm = require('vm');
              'Received type object'
   });
 });
+
+// vm.wrapFunction
+{
+  assert.strictEqual(
```

Comment:
More tests for
- calling the function with arguments and check that the arguments can be used as expected, e.g. can be returned.
- another syntax error with random garbage as source